### PR TITLE
TypeScript: Allow SapperAnchorProps & SvelteKitAnchorProps on svelte:element

### PR DESCRIPTION
**Pull request by pascalpp**
Ref: https://github.com/sveltejs/language-tools/issues/1576

### CODE REVIEW

**Comment by pascalpp** [comment]
@dummdidumm this is my attempt to address [the Typescript issue I opened](#1576). It passes tests, but I'm gonna see if I can figure out how to add a test that fails without these changes.

File: packages/svelte2tsx/test/htmlx2jsx/samples/sveltekit-anchor-attrs/expectedv2.js [hunk 1]
```diff
@@ -1,3 +1,4 @@
  { svelteHTML.createElement("a", {"sveltekit:noscroll":true,}); }
  { svelteHTML.createElement("a", {"sveltekit:prefetch":true,}); }
- { svelteHTML.createElement("a", {"sveltekit:reload":true,}); }
\ No newline at end of file
+ { svelteHTML.createElement("a", {"sveltekit:reload":true,}); }
+ { svelteHTML.createElement("a", { "sveltekit:prefetch":true,}); }
```
Comment (pascalpp) [inline packages/svelte2tsx/test/htmlx2jsx/samples/sveltekit-anchor-attrs/expectedv2.js:4]: not sure why, but this test fails without this extra space inside the brackets.

**Comment by pascalpp** [comment]
I updated the samples for svelte-element and sveltekit-anchor-attrs. I verified that these samples fail in master. Let me know what you think.

**Review by dummdidumm — APPROVED** [review]
Thank you!

**Pull request merged**